.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -1,70 +1,72 @@
 name: 🐛 Bug Report
 description: Create a bug report to help us improve Quire
-labels: [bug]
+labels: ['status:triage needed']
 assignees:
 - erin-cecele
-- geealbers
 body:
 - type: markdown
   attributes:
     value: |
       # Thank you for opening a new issue! 
 - type: checkboxes
   attributes:
-    label: Before proceeding, check to make sure there isn’t an existing issue for this bug.
-    description:
+    label: Before proceeding, make sure there isn’t an existing issue for this bug.
+    description: Check the [Issue Tracker](https://github.com/thegetty/quire/issues) to avoid duplicate issues.
     options:
     - label: I have searched the existing issues and determined this is a new bug.
       required: true    
 - type: textarea
-  id: reproduce
+  id: expected
   attributes:
-    label: Steps to reproduce
-    description: Describe what steps you took that led to the issue. Please be as specific as possible.
+    label: Expected Behavior
+    description: Describe what you expected to happen had everything operated accordingly. 
   validations:
     required: true
 - type: textarea
   id: actual
   attributes:
-    label: Actual behavior
-    description: Describe what happened and how it diverged from what was anticipated.
+    label: Actual Behavior
+    description: Describe what happened.
   validations:
     required: true
 - type: textarea
-  id: expected
+  id: reproduce
   attributes:
-    label: Expected behavior
-    description: Describe what you expected to happen had everything operated accordingly. 
+    label: Steps to Reproduce
+    description: Describe what steps you took that led to the issue. Please be as specific as possible.
+    placeholder: Step 1... Step 2... Step 3... 
   validations:
     required: true
 - type: textarea
-  id: version 
+  id: quire-version 
   attributes:
-    label: Version numbers
-    description: Providing version numbers will help us diagnose the issue more quickly. Run `quire --version` to get the Quire version number and `node --version` to get the Node version number. 
-    placeholder: Please include operating system (macOS/Windows), operating system version, Quire version, Node version, and any other version numbers relevant to the issue.
+    label: Version Numbers
+    description: Run `quire info --debug` and list the version numbers for both your **project** (`quire-cli`, `quire-11ty`, `starter`) and your **system** (`quire-cli`, `node`, `npm`, and `os`).   
```

```suggestion
    description: Run `quire info --debug` from your project root to list the version numbers for both your **project** (`quire-cli`, `quire-11ty`, `starter`) and your **system** (`quire-cli`, `node`, `npm`, and `os`).   
```
Edited to clarify where to run the command from - "root" could also be "directory" or whatever you think would make the most sense to users.

---

Done. Thank you @hbalenda